samples/README.md

```diff
@@ -37,6 +37,7 @@ Here you can find all hosted samples of [Web Chat](https://github.com/microsoft/
 | [`03.speech/e.select-voice`](https://github.com/microsoft/BotFramework-WebChat/tree/master/samples/03.speech/e.select-voice)                                                                                                       | Demonstrates how to select speech synthesis voice based on activity.                                                                                                                                                                | [Select Voice Demo](https://microsoft.github.io/BotFramework-WebChat/03.speech/e.select-voice)                                                                                                                                                       |
 | [`03.speech/f.web-browser-speech`](https://github.com/microsoft/BotFramework-WebChat/tree/master/samples/03.speech/f.web-browser-speech)                                                                                           | Demonstrates how to implement text-to-speech using Web Chat's browser-based Web Speech API. (link to W3C standard in the sample)                                                                                                    | [Web Speech API Demo](https://microsoft.github.io/BotFramework-WebChat/03.speech/f.web-browser-speech)                                                                                                                                               |
 | [`03.speech/g.hybrid-speech`](https://github.com/microsoft/BotFramework-WebChat/tree/master/samples/03.speech/g.hybrid-speech)                                                                                                     | Demonstrates how to use both browser-based Web Speech API for speech-to-text, and Cognitive Services Speech Services API for text-to-speech.                                                                                        | [Hybrid Speech Demo](https://microsoft.github.io/BotFramework-WebChat/03.speech/g.hybrid-speech)                                                                                                                                                     |
+| [`03.speech/h.select-audio-input-device`](https://github.com/microsoft/BotFramework-WebChat/tree/master/samples/03.speech/h.select-audio-input-device)                                                                             | Demonstrates how to select an audio input device.                                                                                                                                                                                   | [Select Audio Input Device Demo](https://microsoft.github.io/BotFramework-WebChat/03.speech/h.select-audio-input-device) [(Comprehensive)](https://microsoft.github.io/BotFramework-WebChat/03.speech/h.select-audio-input-device)                   |
```

Comment:
Do we need / can there be an equivalent for DLS? Might want to indicate in the title/description that it's for CS